src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+
+Examples of sites that may use this API include:
+
+- Note-taking applications where users want to capture handwritten notes and have them translated
+  into text.
+- Forms applications where users use pen or finger input due to time constraints.
+- Games that require filling in letters, such as crosswords or Hangman.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use Handwriting Recognition {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Handwriting Recognition API locally, without an origin trial token, enable
+the `#experimental-web-platform-features` flag in `chrome://flags`.
+
+{% Aside %} Note that the API is currently exclusive to Chrome&nbsp;OS devices. {% endAside %}
+
+> REVIEW: Add the version? Probably M90+
+
+> REVIEW: There is no Origin Trial yet.
+
+### Feature detection
+
+Browser support can be detected by checking for the existence of the `createHandwritingRecognizer()`
+method on the navigator object:
+
+```js
+function supportsHandwritingRecognition() {
+  return 'createHandwritingRecognizer' in navigator;
+}
+```
+
+### Core concepts
+
+> TODO: Image showing the core entities
+
+The Handwriting Recognition API converts handwritten input into text, regardless of the input
+method (mouse, touch, pen). The API has four main entities:
+
+1. A _point_ represents where (i.e., X and Y coordinates) the pointer was at a particular time.
+2. A _stroke_ consists of one or more points. The recording of a stroke starts when the user puts
+   the pointer down (i.e., clicks the primary mouse button, or touches the screen with their pen or
+   finger) and ends when they raise the pointer back up.
+3. A _drawing_ consists of one or more strokes. The actual recognition takes place on this level.
+4. The _recognizer_ is configured with the expected input language. It is used to create an instance of a drawing
+   with the recognizer configuration applied.
+
+#### Creating a recognizer
+
+To recognize text from handwritten input, you need to obtain an instance of a
+`HandwritingRecognizer` by calling the `createHandwritingRecognizer()` method on the navigator
+object, and passing constraints to it. They are used to determine the handwriting recognition model
+that should be used. Currently, you can specify a list of languages in order of preference. In the
+following example, the developer would prefer to recognize texts in English and fall back to German
+when available.
```

Comment:
```suggestion
as spoken in Germany when available.
```

Reply:
(Will this in turn fall back to "de"?)

Reply:
As per the proposal, [it should](https://github.com/WICG/handwriting-recognition/blob/main/explainer.md#language-handling). However, it doesn't seem to be implemented right now (`en` works, but `en-GB` won't).